.github/workflows/ci.yml

```diff
@@ -69,6 +69,7 @@ jobs:
       - name: Run tests
         shell: bash -l {0}
         run: |
+          pip install -e kinfraglib
```

Comment:
Could you add a comment like

```bash
# Install kinfraglib again (comes already with the environment) to ensure this commit's kinfraglib version for CI testing
```

above the `pip` line.

That way we may remember why we do this here again.

Reply:
Maybe _after_ we have verified that this hack actually works :D

Reply:
Meaning - let's wait for the CI run to finish first.